packages/client/src/views/Performance/FieldAgentList.tsx

```diff
@@ -327,6 +329,13 @@ function FieldAgentListComponent(props: IProps) {
     ]
   }
 
+  const getFieldAgentTypeLabel = useCallback(
+    (type) => {
+      return userMessages[type] ? intl.formatMessage(userMessages[type]) : type
+    },
+    [intl]
+  )
+
```

Comment:
I think a plain function would suffice in this case as the function itself isn't passed down as a prop. useCallback is more for the case where you have a `const onClick = (event) =>` type of a handler that is created in the render function and you don't want the receiving component to get a new function reference on every render. Was it eslint that was suggesting this approach?

Reply:
No, I just thought it would assign new function value every render otherwise

Reply:
Okay 👍 In this case it's fine as you're not passing the function reference to any component as a prop

Reply:
Understood @rikukissa, thanks very much. Doing the changes